pkg/lightning/config/config.go

```diff
@@ -1101,6 +1102,7 @@ type TikvImporter struct {
 	StoreWriteBWLimit       ByteSize `toml:"store-write-bwlimit" json:"store-write-bwlimit"`
 	LogicalImportBatchSize  ByteSize `toml:"logical-import-batch-size" json:"logical-import-batch-size"`
 	LogicalImportBatchRows  int      `toml:"logical-import-batch-rows" json:"logical-import-batch-rows"`
+	LogicalImportPrepStmt   bool     `toml:"logical-import-prep-stmt" json:"logical-import-prep-stmt"`
```

Comment:
please add a integration test for it

Reply:
does the test of TestAdjustLogicalImportPrepStmt look good to you?

Reply:
ut it not enough for new features that's visible to user